infrastructure/src/main/java/org/corfudb/infrastructure/RemoteMonitoringService.java

```diff
@@ -93,6 +93,8 @@
 
     private final HealingAgent healingAgent;
 
+    private final FailuresAgent failuresAgent;
+
     private final FailureDetectorDataStore fdDataStore;
```

Comment:
Does this need to be a field of `RemoteMonitoringService`? It looks like it's only needed to initialize other components.

Reply:
Yes, I'd like to have as a field, also it's used in two places in RMS, so we need to have it as a field